packages/hint-compat-api/README.md

```diff
@@ -12,6 +12,8 @@ are compatible with the target browsers that you want to support.
 
 * [compat-api-css][compat-api-css]
 * [compat-api-css-next][compat-api-css-next]
+* [compat-api-html][compat-api-html]
+* [compat-api-html-next][compat-api-html-next]
```

Comment:
I think you are missing the link to `architecture`. Also, the images links in architecture are not parsed very well in the website, so I think you can fix that in this PR too.